### ISSUE TITLE
KeyError: None--when running pyscf/future/dmrgscf/test/test_chemps1_n2.py

### ISSUE THREAD

**Issue by dingld** [sunqm/pyscf#15]
File "/home/ding/github-soft/pyscf/future/dmrgscf/chemps2.py", line 72, in kernel
    groupNumber = GROUPNAME_ID[self.groupname]
KeyError: None

The pyscf and Chemps2 are able to run tests separately, but the interface in pyscf failed on my computer. (latest commit 5917df7b8c).

**Comment by sunqm** [comment]
Fixed in latest commit.

**Comment by dingld** [comment]
New error was complained about in the lattest commit.

File "/home/ding/github-soft/pyscf/future/dmrgscf/chemps2.py", line 78, in kernel
    numpy.asarray(self.orbsym))
  File "PyCheMPS2.pyx", line 64, in PyCheMPS2.PyHamiltonian.__cinit__ (PyCheMPS2.cpp:2536)
ValueError: Buffer dtype mismatch, expected 'int' but got 'long'

**Comment by dingld** [comment]
The ValueError can be issued by manually transforming datatype with numpy.int32() on my machine. And then I came across TypeError

 File "/home/ding/github-soft/pyscf/future/dmrgscf/chemps2.py", line 150, in make_rdm1
    return self.make_rdm12(fcivec, norb, nelec, link_index, **kwargs)[0]
TypeError: make_rdm12() takes exactly 4 arguments (5 given)

I tried deleting the argument link_index as in   
return self.make_rdm12(fcivec, norb, nelec, link_index, **kwargs)[0],
 to find the tests end with OK.

I  am not sure if there will be another error actually.
